frontend/src/types.ts

```diff
@@ -1836,14 +1836,19 @@ export interface DatedAnnotationType extends Omit<AnnotationType, 'date_marker'>
 
 export enum ChartDisplayType {
     ActionsLineGraph = 'ActionsLineGraph',
-    ActionsLineGraphCumulative = 'ActionsLineGraphCumulative',
+    ActionsBar = 'ActionsBar',
     ActionsAreaGraph = 'ActionsAreaGraph',
-    ActionsTable = 'ActionsTable',
+    ActionsLineGraphCumulative = 'ActionsLineGraphCumulative',
+    BoldNumber = 'BoldNumber',
     ActionsPie = 'ActionsPie',
-    ActionsBar = 'ActionsBar',
     ActionsBarValue = 'ActionsBarValue',
+    ActionsTable = 'ActionsTable',
     WorldMap = 'WorldMap',
-    BoldNumber = 'BoldNumber',
+}
+export enum ChartDisplayCategory {
```

Comment:
Added `ChartDisplayCategory` as an abstraction over the specific display types

I also wanted to use it in the backend, the way we reuse `ChartDisplayType` thanks to `schema.{ts,json,py}`, but unfortunately `ts-json-schema-generator` doesn't support re-exporting, so we can't do that (unless we start using `ChartDisplayCategory` in a type defined in `schema.ts` itself). I put out a [fix for the library](https://github.com/vega/ts-json-schema-generator/issues/567) (interesting opportunity to learn more about TS syntax internals), but in the meantime it's fine for this to be frontend-only

Reply:
`ChartDisplayCategory` would specifically be super useful for caching